src/frontend/js/pages/TeacherDashboardCoursesLoader/index.spec.tsx

```diff
@@ -4,6 +4,7 @@ import { render, screen } from '@testing-library/react';
 import fetchMock from 'fetch-mock';
 import { IntlProvider } from 'react-intl';
 
+import { CunninghamProvider } from '@openfun/cunningham-react';
```

Comment:
This had not been done in #2106 ?

Reply:
There what's happen on the course page, look's ok to me:
![image](https://github.com/openfun/richie/assets/139848/2b8420eb-aafc-40a5-bbe3-418d5de516c2)

Reply:
in #2106 we bump cunningham to the last version
Here we've add a cunningham select in our sidebar, so cunningham provider is now needed.